ci/ansible/roles/pulp/tasks/pulp_server.yaml

```diff
@@ -16,9 +16,12 @@
     - qpid-cpp-server
     - qpid-cpp-server-linearstore
 
-- name: Install package for migration 0009
-  action: "{{ ansible_pkg_mgr }} name=qpid-tools state=latest"
-  when: ansible_distribution == "RedHat" and ansible_distribution_major_version|int == 7
+- name: Install packages for migration 0009
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=latest"
+  when: ansible_distribution == "RedHat" and ansible_distribution_major_version|int >= 6
+  with_items:
+    - python-qpid-qmf
+    - qpid-tools
```

Comment:
How come you only want to install these packages for RHEL 6 and 7? What installs them for Fedora?